containers/proxy-squid-image/Dockerfile

```diff
@@ -65,4 +65,4 @@ EXPOSE 8080/tcp
 
 VOLUME [ "/etc/uyuni", "/var/cache/squid" ]
 
-CMD uyuni-configure.py && squid -z --foreground && squid -FC -d 1 --foreground
+CMD configure_and_run.py
```

Comment:
I've decided to wrap the call to squid into the py script. It's the easiest way to set the `--enable-delay-pools` flag conditionally without messing with env variables etc.